src/pages/cookie-settings.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: "Cookie Settings"
+date: "2022-01-25"
+---
+
+import ManageCookieSettings from '@site/src/components/ManageCookieSettings'
+
+# Cookie settings
+
+### About this cookie policy
+
+This Cookie Policy explains what cookies are and how we use them. You should read this policy to understand what cookies are, how we use them, the types of cookies we use i.e, the information we collect using cookies and how that information is used and how to control the cookie preferences. For further information on how we use, store and keep your personal data secure, see our Privacy Policy.
```

Comment:
Link to Privacy Policy (https://snowplowanalytics.com/privacy-policy/). I know it does not match 100% but it’s probably good enough